### ISSUE TITLE
NPE Creating Named Entity

### ISSUE THREAD

**Issue by kolinkrewinkel** [kumpelblase2/Remote-Entities#74]
```
[20:44:04] [Server thread/WARN]: java.lang.NullPointerException
[20:44:04] [Server thread/WARN]:    at net.minecraft.server.v1_7_R1.PlayerConnection.isDisconnected(PlayerConnection.java:1793)
[20:44:04] [Server thread/WARN]:    at org.bukkit.craftbukkit.v1_7_R1.entity.CraftPlayer.teleport(CraftPlayer.java:428)
[20:44:04] [Server thread/WARN]:    at org.bukkit.craftbukkit.v1_7_R1.entity.CraftEntity.teleport(CraftEntity.java:200)
[20:44:04] [Server thread/WARN]:    at de.kumpelblase2.remoteentities.entities.RemotePlayer.spawn(RemotePlayer.java:66)
[20:44:04] [Server thread/WARN]:    at de.kumpelblase2.remoteentities.ChunkEntityLoader.spawn(ChunkEntityLoader.java:127)
[20:44:04] [Server thread/WARN]:    at de.kumpelblase2.remoteentities.ChunkEntityLoader.queueSpawn(ChunkEntityLoader.java:117)
[20:44:04] [Server thread/WARN]:    at de.kumpelblase2.remoteentities.EntityManager.createNamedEntity(EntityManager.java:203)
```

1.7 dev build.

**Comment by kumpelblase2** [comment]
Did you try it after https://github.com/kumpelblase2/Remote-Entities/commit/5b0205c5ab62e70635378075125a0cf2c26d17b0 this commit?

**Comment by kolinkrewinkel** [comment]
Yeah. The commit fixed the infinite loop but not (this) NPE.

**Comment by kumpelblase2** [comment]
Hm interesting, it worked for me at that point... I'll see what I can do.

**Comment by kumpelblase2** [comment]
This should've been closed a while ago by pr #75 so I'm closing this now.
